src/components/elementView/elementTable.tsx

```diff
@@ -64,7 +67,8 @@ interface ElementTableState {
   fields: Array<string>;
   isEditFormOpen: boolean;
   editFormValues: ElementObjectType;
-  isDeleteOpen: boolean;
+  openAlert: ElementTableOpenAlert;
```

Comment:
other option would have been to have 3 bools on state (isDeleteOpen, isGrantAdminOpen, isRevokeAdminOpen) to handle all alerts, which felt gross!

Reply:
trueeee